packages/atomic/src/components/commerce/atomic-commerce-breadbox/atomic-commerce-breadbox.tsx

```diff
@@ -0,0 +1,420 @@
+  }
+
+  private updateShowLessDisplay() {
+    this.show(this.showLess);
+    if (this.showLess.offsetTop === 0) {
+      this.hide(this.showLess);
+    }
+  }
+
+  private adaptBreadcrumbs() {
+    if (!this.breadcrumbs.length) {
+      return;
+    }
+    this.showAllBreadcrumbs();
+
+    if (!this.isCollapsed) {
+      this.updateShowLessDisplay();
+      return;
+    }
+
+    this.updateShowMoreValue(this.breadcrumbs.length);
+    this.hideOverflowingBreadcrumbs();
+  }
+
+  private get isOverflowing() {
+    const listElement = this.host.shadowRoot!.querySelector('ul');
+    if (!listElement) {
+      return false;
+    }
+    return listElement.scrollWidth > listElement.clientWidth;
+  }
+
+  private updateShowMoreValue(value: number) {
+    this.numberOfCollapsedBreadcrumbs = value;
+    if (value === 0) {
+      this.hide(this.showMore);
+      return;
+    }
+    this.show(this.showMore);
+    this.showMore.textContent = `+ ${value.toLocaleString(
+      this.bindings.i18n.language
+    )}`;
+    this.showMore.setAttribute(
+      'aria-label',
+      this.bindings.i18n.t('show-n-more-filters', {
+        value,
+      })
+    );
+  }
+
+  private valueForFacetType = (
+    type: string,
+    field: string,
+    value: BreadcrumbValue<AnyFacetValue>
+  ): string => {
+    switch (type) {
+      case 'numericalRange':
+        return this.bindings.store.state.numericFacets[field].format(
+          value.value as NumericFacetValue
+        );
+      case 'dateRange':
+        return this.bindings.store.state.dateFacets[field].format(
+          value.value as DateFacetValue
+        );
+      case 'hierarchical':
+        return getFieldValueCaption(
+          field,
+          (value.value as CategoryFacetValue).value,
+          this.bindings.i18n
+        );
+      default:
+        return getFieldValueCaption(
+          field,
+          (value.value as RegularFacetValue).value,
+          this.bindings.i18n
+        );
+    }
+  };
+
+  private buildBreadcrumb(breadcrumb: Breadcrumb<AnyFacetValue>) {
+    let facetStateName:
+      | 'facets'
+      | 'categoryFacets'
+      | 'numericFacets'
+      | 'dateFacets';
+
+    switch (breadcrumb.type) {
+      case 'hierarchical':
+        facetStateName = 'categoryFacets';
+        break;
+      case 'numericalRange':
+        facetStateName = 'numericFacets';
+        break;
+      case 'dateRange':
+        facetStateName = 'dateFacets';
+        break;
+      default:
+        facetStateName = 'facets';
+    }
+
+    return breadcrumb.values.map((value) => {
+      return {
+        facetId: breadcrumb.facetId,
+        label:
+          this.bindings.store.state[facetStateName][
+            breadcrumb.facetId
+          ]?.label(),
+        deselect: value.deselect,
+        formattedValue: [
+          this.valueForFacetType(
+            breadcrumb.type,
+            breadcrumb.facetId,
+            value as BreadcrumbValue<AnyFacetValue>
+          ),
+        ],
```

Comment:
The bug referenced in my other comment is caused by this part of the code. You need to map all the `path` values together to form `formattedValue`

<img width="492" alt="image" src="https://github.com/coveo/ui-kit/assets/78121423/19c3812e-491f-4d0f-a76c-62fb3c1aae48">